server/services/config/conditions/standardConditions/v2/standardConditions.ts

```diff
@@ -0,0 +1,31 @@
+import { StandardCondition } from '../../../../../data/licenceClientTypes'
+
+export const standardConditions: StandardCondition[] = [
+  {
+    text: 'Be of good behaviour and not behave in a way which undermines the purpose of the licence period.',
+  },
+  {
+    text: 'Not commit any offence.',
+  },
+  {
+    text: 'Keep in touch with the supervising officer in accordance with instructions given by the supervising officer.',
+  },
+  {
+    text: 'Receive visits from the supervising officer in accordance with instructions given by the supervising officer.',
+  },
+  {
+    text: 'Reside permanently at an address approved by the supervising officer and obtain the prior permission of the supervising officer for any stay of one or more nights at a different address.',
+  },
+  {
+    text: 'Not undertake work, or a particular type of work, unless it is approved by the supervising officer and notify the supervising officer in advance of any proposal to undertake work or a particular type of work.',
+  },
+  {
+    text: 'Not travel outside the United Kingdom, the Channel Islands or the Isle of Man except with the prior permission of your supervising officer or for the purposes of immigration deportation or removal.',
+  },
+  {
+    text: 'Tell your supervising officer if you use a name which is different to the name or names which appear on your licence.',
```

Comment:
Not entirely sure where these should of gone, so just stuck them at the bottom. Happy to move them around if we know what the order is?